model/user.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, String, Integer
+Base = declarative_base()
+
+class user(Base):
```

Comment:
use CapWords convention for class names   so :     user --> User
https://www.python.org/dev/peps/pep-0008/#class-names